src/pages/Account/Profile.tsx

```diff
@@ -71,19 +71,19 @@ const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
               placeholder="Your display name"
               ref={register({
                 required: true,
-                maxLength: 16,
+                maxLength: 30,
               })}
               autoComplete="off"
             />
             <span className="input-info">
-              This is your public name (max 16 characters)
+              This is your public name (max 30 characters)
             </span>
             {errors.partyName && errors.partyName.type === "required" && (
               <span className="input-error">Display name is required</span>
             )}
             {errors.partyName && errors.partyName.type === "maxLength" && (
               <span className="input-error">
-                Display name must be 16 characters or less
+                Display name must be 30 characters or less
               </span>
             )}
```

Comment:
@sparkletown/product @sparkletown/design @mike-lvov Should we align the text here so that they both say "Display name" rather than one of them being "public name"?

Reply:
Yes, `Display name` works. + this should be imported from `settings`

Reply:
> + this should be imported from settings

https://github.com/sparkletown/sparkle/pull/1379#discussion_r630702721 ;)

Reply:
> Yes, `Display name` works for me. + this should be imported from `settings`

Fixed